string piece_type
niryo_robot_msgs/ObjectPose pose
float32 confidence
int32 pixel_x
int32 pixel_y
